### PR TITLE
Bump kops

### PR DESCRIPTION
**Pull request by bhavi-koduru**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: development/kops/set_environment.sh [hunk 1]
```diff
@@ -24,10 +24,10 @@ export NODE_INSTANCE_TYPE=${NODE_INSTANCE_TYPE:-t3.medium}
 export NODE_ARCHITECTURE=${NODE_ARCHITECTURE:-amd64}
 export UBUNTU_RELEASE=${UBUNTU_RELEASE:-focal-20.04}
 export IPV6=${IPV6:-false}
-if [ "$RELEASE_BRANCH" == "1-29" ]; then
-	export KOPS_VERSION="1.29.0-alpha.3"
+if [ "$RELEASE_BRANCH" == "1-30" ]; then
```
Comment (rcrozean) [inline development/kops/set_environment.sh:27]: We will want to continue using this for 1-29 as well. We could do something line 
`if [ "${RELEASE_BRANCH##*}- >= 29`

File: development/kops/set_environment.sh [hunk 2]
```diff
@@ -24,10 +24,10 @@ export NODE_INSTANCE_TYPE=${NODE_INSTANCE_TYPE:-t3.medium}
 export NODE_ARCHITECTURE=${NODE_ARCHITECTURE:-amd64}
 export UBUNTU_RELEASE=${UBUNTU_RELEASE:-focal-20.04}
 export IPV6=${IPV6:-false}
-if [ "$RELEASE_BRANCH" == "1-29" ]; then
-	export KOPS_VERSION="1.29.0-alpha.3"
+if [ "$RELEASE_BRANCH" >= "1-30" ]; then
```
Comment (rcrozean) [inline development/kops/set_environment.sh:27]: We want this to be 29 or higher. Also not sure the string check will work that way. We want to strip the minor version off.

**Review by rcrozean — CHANGES_REQUESTED** [review]

File: development/kops/set_environment.sh (hunk 2 above)
Comment (bhavi-koduru) [reply development/kops/set_environment.sh]: Verified from the logs that RELEASE_BRANCH takes the string as 1.30
`++ export RELEASE_BRANCH=1-30
++ RELEASE_BRANCH=1-30`
modified the comparator check

**Review by rcrozean — APPROVED** [review]

**Pull request merged**